projects/components/src/overlay/sheet/sheet-overlay.component.ts

```diff
@@ -61,6 +67,7 @@ export class SheetOverlayComponent {
   public readonly closeOnEscape: boolean;
   public readonly attachedTriggerTemplate?: TemplateRef<unknown>;
   public isViewCollapsed: boolean;
+  public navigationParams: ExternalNavigationParams | undefined;
```

Comment:
This should be `InAppNavigationParams`. Or you can keep just `NavigationParams`

Reply:
@anandtiwary open in new tab supports only external nav params. Hence the choice of types.

Alternative would be to use open in new tab conditionally if nav type is internal. But that can be added in a future iteration if we have a case for it. Currently all our hyperlinking cases from sheets are for opening in new tabs / external.

Reply:
But we might be opening an internal link right? So, can we update open-in-new-tab to accept  `NavigationParam` ?

Reply:
@anandtiwary Internal nav links are also converted to ExternalNavParams to OpenInNewTab currently.

If we want to extend support for InternalNavParams by default in OpenInNewTab, we can add support for that separately IMO. If you'd like that to be added, I can take that up after this.

Reply:
Okay. Sure. We can do it follow up